### PR TITLE
Feature/set arm to height command

### PR DESCRIPTION
**Pull request by JohnGilb**

### CODE REVIEW

File: CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    private static Logger log = Logger.getLogger(SetArmToAngleCommand.class);
+    
+    @Inject
+    public SetArmToAngleCommand(ArmTargetSubsystem armTarget) {
+        this.armTarget = armTarget;
+    }
+    
+    public void setGoalAngle(double angle) {
+        
+        double minValue = 0;
```
Comment (aschokking) [inline CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java]: at least the max should probably be a shared property somewhere eventually

File: CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Override
+    public void execute() {
+        // nothing happens here.
+    }
+    
+    @Override
+    public boolean isFinished() {
+        return true;
```
Comment (aschokking) [inline CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java]: wouldn't we want this command to potentially wait until the angle is achieved to end?

File: CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+public class SetArmToAngleCommand extends BaseCommand{
+
+    protected ArmTargetSubsystem armTarget;
+    protected double goalAngle;
+    
+    private static Logger log = Logger.getLogger(SetArmToAngleCommand.class);
+    
+    @Inject
+    public SetArmToAngleCommand(ArmTargetSubsystem armTarget) {
+        this.armTarget = armTarget;
```
Comment (aschokking) [inline CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java]: this should probably in principal require the target subsystem?

File: CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java (hunk 2 above)
Comment (JohnGilb) [inline CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java]: I wrote this command with a lot of protected variables so we could easily subclass it for a "SetArmToAngleAndWait" command

File: CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java (hunk 3 above)
Comment (JohnGilb) [inline CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java]: Makes sense

File: CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java (hunk 1 above)
Comment (JohnGilb) [inline CompetitionTemplate/src/competition/subsystems/arm/arm_commands/SetArmToAngleCommand.java]: agreed

**Comment by aschokking** [comment]
lgtm

**Pull request merged**